src/actions/signup-modal/signup-modal-actions.spec.js

```diff
@@ -0,0 +1,18 @@
+import Actions from '../../reducers/actions';
+import { closeSignupModal, swapModal } from '.';
+
```

Comment:
extra newline (we should probably look into eslint rules for these...)

Reply:
Ya its wierd because it will auto correct that in a regular js file but not the spec file. I removed the new line though